### ISSUE TITLE
How to configure webpack yourself

### ISSUE THREAD

**Issue by sowdf** [developit/preact#1224]
How to configure webpack yourself

**Comment by stefvw93** [comment]
I don't think this has anything to do with preact in particular. There's literally hundreds of tutorials on the web, use google.
